tsconfig.json

```diff
@@ -8,7 +8,8 @@
     "skipLibCheck": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
-    "sourceMap": true
+    "sourceMap": true,
+    "forceConsistentCasingInFileNames": true
```

Comment:
why?

Reply:
prevents you from strange names like _strangeFILE_.ts